### PR TITLE
chore(shell): make shell script without preinstall jq package

### PR DESCRIPTION
**Pull request by neilkuan**
chore(shell): make shell script without preinstall jq package
截圖 2020-10-19 上午8 59 33

### CODE REVIEW

File: 01-install-eks-cluster/install.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -e
 set -x
 
 sudo apt install jq -y
```
Comment (RicoToothless) [inline 01-install-eks-cluster/install.sh:6]: Please delete this command as well

**Review by RicoToothless — COMMENTED** [review]
Did you delete your forked repository?
Because of your pull request displays `unknown repository`.
issue https://github.com/isaacs/github/issues/168

File: 01-install-eks-cluster/install.sh (hunk 1 above)
Comment (neilkuan) [reply 01-install-eks-cluster/install.sh:6]: okay

**Comment by neilkuan** [comment]
made some mistakes , sorry ~ .I will close this PR and create another PR for this .

**Pull request closed without merging**